demo/index.tsx

```diff
@@ -19,24 +19,40 @@ export default class Demo extends React.Component {
     introspection: defaultPreset,
   };
 
+  // not part of state because we parse it in constructor and just use in render
+  displayOptions = {};
+
   constructor(props) {
     super(props);
 
-    const { url, withCredentials } = getQueryParams();
+    const { url, withCredentials, ...restQueryParams } = getQueryParams();
+    // parse rest of query params we expect them to be in JSON format
+    // we might pick also some options that are not valid displayOptions but they will be ignored
+    // by Voyager component
+    for (const [key, value] of Object.entries(restQueryParams)) {
+      try {
+        this.displayOptions[key] = JSON.parse(value);
+      } catch (_) {
+        console.log(
+          'Not expected value for key "',
+          key,
+          '" value >',
+          value,
+          '< it is not in json format so just ignoring it',
+        );
+      }
+    }
     if (url) {
-      this.state.introspection = introspectionQuery => fetch(url, {
-        method: 'post',
-        headers: {
-          'Accept': 'application/json',
-          'Content-Type': 'application/json',
-        },
-        body: JSON.stringify({ query: introspectionQuery }),
-        ...(
-          withCredentials === 'true'
-            ? { credentials: 'include', mode: 'cors' }
-            : {}
-        ),
-      }).then(response => response.json());
+      this.state.introspection = introspectionQuery =>
+        fetch(url, {
+          method: 'post',
+          headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({ query: introspectionQuery }),
+          ...(withCredentials === 'true' ? { credentials: 'include', mode: 'cors' } : {}),
+        }).then(response => response.json());
```

Comment:
Can you please run prettier on these files as a separate PR?
It would way easier to review this PR and will keep commit history clean.

Reply:
Created PR https://github.com/APIs-guru/graphql-voyager/pull/143 and will rebase this it after it will be merged

Reply:
rebased